mitodl/mitxpro-theme
# style: update favicon

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/3409



### Description (What does it do?)
- Updates the favicon


### How can this be tested?
Setup this theme in your edX instance, and open any legacy page e.g. instructor dashboard and check that you see the updated favicon.
